### ISSUE TITLE
NTR: hemorrhagic gastroenteritis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10618]
**Preferred term label:**
hemorrhagic gastroenteritis

**Synonyms**
bacterial hemorrhagic enterocolitis

**Definition (free text, please give PubMed ID)**
https://www.healthline.com/health/acute-hemorrhagic-gastroenteritis-in-humans; PMID: 12640523; 

Hemorrhagic gastroenteritis is a condition where an individual experiences bloody, watery stools and crampy abdominal pains. It is typically a food-borne disease that occurs due to the contamination of foods by infectious organisms such as bacteria (mostly Enterohemorrhagic Escherichia coli, Salmonella or Shigella), viruses (such as rotaviruses or noroviruses), or parasites (such as Giardia). Besides, hemorrhagic gastroenteritis can also be induced by medications (like steroid-therapy and antibiotics), or certain chemicals, like (lead poising, copper sulfate poising

**Parent term (use hpo.jax.org/app)**
Gastrointestinal hemorrhage HP:0002239

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158
